src/contexts/NotificantionManager/NotificationComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import { Toast, ToastVariant } from '@chili-publish/grafx-shared-components';
+import { NotificationWrapper } from './Notification.styles';
+import { useNotificationManager } from './NotificationManagerContext';
+
+const DEFAULT_NOTIFICATION_DURATION = 5000;
+
+function NotificationComponent() {
+    const { currentNotification, removeNotification } = useNotificationManager();
```

Comment:
can we propagate it via props?

Reply:
done 573926c86b8b1e62dade77fa67e287d954d8d93d